src/api/language.js

```diff
@@ -24,6 +25,20 @@ const loadMessages = (lang) => {
     .then((messages) => Object.assign.apply({}, messages))
 }
 
+const loadAllLanguagesMessages = () => {
+  const messages = {}
+  allLanguages.forEach(language => {
+    try {
+      messages[language.lang] = require(`../../i18n/${language.lang}.json`)
+    } catch (error) {
+      // Translation not available, discard
+    }
+  })
+
+  return messages
+}
+
 export {
-  loadMessages
+  loadMessages,
```

Comment:
Is `loadMessages` still useful?

With all translations loaded from the start, the language selection component shouldn't have to explicitly anything anymore. Is there something that could be cleaned up there?

Reply:
yeah, this can be removed. 

so we already load all the language, we just need to `i18n.setLocale()` to set the new locale, and no more `loadMessages` needed.

Reply:
after this has been removed, we can remove `load` method then